packages/teleport/src/Databases/DatabaseList/DatabaseList.tsx

```diff
@@ -29,16 +29,22 @@ import {
 import Table from 'design/DataTable/Paged';
 import isMatch from 'design/utils/match';
 import InputSearch from 'teleport/components/InputSearch';
-import { Database } from 'teleport/services/databases';
+import { Database, DbProtocol } from 'teleport/services/databases';
+import ConnectDialog from 'teleport/Databases/ConnectDialog';
 
 function DatabaseList(props: Props) {
-  const { databases = [], pageSize = 20 } = props;
+  const { databases = [], pageSize = 20, user, clusterId } = props;
 
-  const [sortDir, setSortDir] = React.useState<Record<string, string>>({
+  const [sortDir, setSortDir] = useState<Record<string, string>>({
     name: SortTypes.DESC,
   });
 
-  const [searchValue, setSearchValue] = React.useState<string>('');
+  const [searchValue, setSearchValue] = useState<string>('');
+
+  const [dbConnectInfo, setDbConnectInfo] = useState<{
+    name: string;
+    protocol: DbProtocol;
+  }>(undefined);
```

Comment:
i'd use `null` when we are intentionally wanting to assign something as no value